modules/dxp/apps/commerce-avalara/commerce-avalara-connector-api/src/main/java/com/liferay/commerce/avalara/connector/configuration/CommerceAvalaraConnectorConfiguration.java

```diff
@@ -16,18 +16,13 @@
 
 import aQute.bnd.annotation.metatype.Meta;
 
-import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
-
 /**
- * @author Calvin Keum
+ * @author Katie Nesterovich
  */
-@ExtendedObjectClassDefinition(
```

Comment:
Here i think we need to add the annotation for having system scope
ExtendedObjectClassDefinition.Scope.SYSTEM

Reply:
Hmm, if I add the annotation, then it appears in the System Configuration (currently we have that happen with `@ExtendedObjectClassDefinition(
	category = "tax-rate", scope = ExtendedObjectClassDefinition.Scope.GROUP
)`

And it's displayed in both system and instance settings. 
![image](https://user-images.githubusercontent.com/51712906/103031318-36c53d80-4523-11eb-8db1-783f9e092bfb.png)

![image](https://user-images.githubusercontent.com/51712906/103031360-4c3a6780-4523-11eb-9b59-13b488fc22ba.png)

![image](https://user-images.githubusercontent.com/51712906/103031379-55c3cf80-4523-11eb-9107-e56d03baa50a.png)

Do we want that? 
If yes, should add it. If no, should remove from the channel configuration too.